### PR TITLE
Layout changes

### PR DESCRIPTION
**Pull request by MelGGit**
I made several adjustments to the Layout of my first and second User Story. Also i added react router and implemented some logic to show a specific detail page on click on a podcast. Lastly i created a DetailsPage. I took my details code which i had in the Card itself and put it in the new page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+import { Route, Switch, useHistory } from 'react-router-dom'
+import DetailsPage from './pages/DetailsPage/DetailsPage'
+import { useState } from 'react'
 
 function App() {
-  return <PodcastListPage podcasts={podcasts} />
+  const [detailedPodcast, setDetailedPodcast] = useState()
+  const { push } = useHistory()
+  return (
+    <>
```
Comment (Morten-Moeller) [inline src/App.js:12]: You have the Switch wrapping everything you don't need <> </>

File: src/components/PodcastCard/PodcastCard.js [hunk 2]
```diff
@@ -4,43 +4,86 @@ import PropTypes from 'prop-types'
...
+  grid-template-columns: 3rem 1fr 1rem;
+  height: 3rem;
+  gap: 0.5rem;
+  width: 100%;
+  padding: 0 1.5rem 0 1rem;
+`
+const ImageContainer = styled.div`
+  height: 3rem;
+  width: 3rem;
+  align-self: flex-start;
```
Comment (Morten-Moeller) [inline src/components/PodcastCard/PodcastCard.js:51]: ImageContainer is a child of a grid right? So `start` would be the correct value for `align-self`
[https://css-tricks.com/snippets/css/complete-guide-grid/](https://css-tricks.com/snippets/css/complete-guide-grid/)

File: src/pages/DetailsPage/DetailsPage.spec.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+import DetailsPage from './DetailsPage'
+  
+describe('DetailsPage', () => {
+    it('renders', () => {
+        render(<DetailsPage />)
+        expect(screen.getByText('DetailsPage')).toBeInTheDocument()
```
Comment (Morten-Moeller) [inline src/pages/DetailsPage/DetailsPage.spec.js:7]: This looks like you forget the test?

File: src/components/PodcastCard/PodcastCard.js [hunk 4]
```diff
@@ -4,43 +4,86 @@ import PropTypes from 'prop-types'
 PodcastCard.propTypes = {
   image: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
+  author: PropTypes.string.isRequired,
```
Comment (Morten-Moeller) [inline src/components/PodcastCard/PodcastCard.js:7]: Can't find onClickDetails and id

**Review by Morten-Moeller — COMMENTED** [review]
Overall nice work! 👾

File: src/pages/DetailsPage/DetailsPage.spec.js (hunk 3 above)
Comment (MelGGit) [reply src/pages/DetailsPage/DetailsPage.spec.js:7]: Ty Done

**Pull request merged**